akka-docs-new/src/test/java/jdocs/tutorial_2/DeviceInProgress.java

```diff
@@ -0,0 +1,26 @@
+package jdocs.tutorial_2;
+
+import java.util.Optional;
+
+import jdocs.tutorial_2.Device.ReadTemperature;
+import jdocs.tutorial_2.Device.RecordTemperature;
+import jdocs.tutorial_2.Device.RespondTemperature;
+import jdocs.tutorial_2.Device.TemperatureRecorded;
+
+class DeviceInProgress1 {
+
+    //#read-protocol-1
+    final static class ReadTemperature {
```

Comment:
I don't have a solution, but since we use the outer class to namespace which step (and have more than one class in the same place) and then only include the inner classes in the sample snippet, if the reader copy-paste it, it won't compile right? should we have separate packages instead to avoid that?